frontend/src/metabase/parameters/components/ParameterValuePicker/ListPickerConnected/ListPickerConnected.unit.spec.tsx

```diff
@@ -0,0 +1,434 @@
+/* eslint-disable jest/no-conditional-expect */
+/* eslint-disable testing-library/no-unnecessary-act */
+import { act, render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
```

Comment:
Thank you, fixed that. It is because for some reason, when you make a model from a GUI question, it allows you to pick a column that obviously isn't a `text` type (true and false as values). I am not sure what the root cause is, but at least I added support for non-string parameters.